androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/ui/currentpatients/CurrentPatientsData.kt

```diff
@@ -0,0 +1,12 @@
+package com.EENX15_22_17.digital_journal.android.ui.currentpatients
+
+// Temporary dataclass before backend is implemented
+data class CurrentPatientsData(
+    val patientName: String,
```

Comment:
´patient´-prefix is unnecessary verbosity here. Use-site is usually `patients.forEach { patient -> patient.patientName }` or similar.

Reply:
Fixed